Sample/SextantSample.UWP/Views/GreenView.xaml.cs

```diff
@@ -0,0 +1,36 @@
+﻿using ReactiveUI;
+using SextantSample.Core.ViewModels;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace SextantSample.UWP.Views
+{
+    /// <summary>
+    /// An empty page that can be used on its own or navigated to within a Frame.
+    /// </summary>
+    public sealed partial class GreenView : Page, IViewFor<GreenViewModel>
+    {
+        public GreenView()
+        {
+            this.InitializeComponent();
+            this.WhenActivated(d =>
```

Comment:
Need `WhenActivated`?

Reply:
Guess not.  This was just drilled into me from the docs...

Reply:
Maybe I will just bind them directly in Xaml using {x:Bind} since UWP is good for that.